py/h2o_wave/hatch_build.py

```diff
@@ -26,6 +26,10 @@ def initialize(self, _version, build_data):
         if not platform:
             # Create a default metadata file in case of noarch builds.
             create_metadata_file('linux', 'amd64')
+            # If conda build, copy binaries into package.
```

Comment:
I though conda has its own build pipeline and doesn't use hatch. How does this work?